ios/test_app.rb

```diff
@@ -116,7 +114,7 @@ def use_react_native!(project_root, target_platform)
   include_react_native!(react_native: react_native.relative_path_from(project_root).to_s,
                         target_platform: target_platform,
                         project_root: project_root,
-                        use_flipper: @use_flipper)
+                        use_flipper: @use_flipper.nil? ? {} : @use_flipper)
```

Comment:
I don’t quite understand the intent here. Passing `{}` means the `if target_platform == :ios && use_flipper` check in `include_react_native!` will pass if `@use_flipper` is not explicitly set. If that’s the case, why not use `true` instead of `{}`? And additionally, how does this handle the conditional default enabling outlined in your PR description?

Reply:
I think this will make more sense if we rename `@use_flipper` -> `@flipper_versions`. The parameter is forwarded to `add_flipper_pods!` in `ios/use_react_native-0.62.rb`. This parameter is not used in 0.60 and 0.61, so Flipper wouldn't be enabled. In 0.63, it will be forwarded to `use_flipper!` from upstream.

```ruby
def add_flipper_pods!(versions = {})
  versions['Flipper'] ||= '~> 0.33.1'
  versions['DoubleConversion'] ||= '1.1.7'
  versions['Flipper-Folly'] ||= '~> 2.1'
  versions['Flipper-Glog'] ||= '0.3.6'
  versions['Flipper-PeerTalk'] ||= '~> 0.0.4'
  versions['Flipper-RSocket'] ||= '~> 1.0'
  ...
```

Reply:
Ahh, gotcha 👍